manager/eris-mint/evm/vm.go

```diff
@@ -565,13 +565,17 @@ func (vm *VM) call(caller, callee *Account, code, input []byte, value int64, gas
 			}
 			acc := vm.appState.GetAccount(addr)
 			if acc == nil {
-				return nil, firstErr(err, ErrUnknownAddress)
+				if _, ok := registeredNativeContracts[addr]; !ok {
```

Comment:
Thus far we have not returned a code size for native contracts.  Is this motivated by a change in solidity, where it newly checks the code length before going to a contract.  We have not had a problem before calling our snative contracts.  Can you document and elaborate?